### DISCUSSION TITLE
Does GSConnect support Virtual monitor

### DISCUSSION THREAD

**Discussion by nose-gnome** [andyholmes/gnome-shell-extension-gsconnect#1784] [Q&A]
I was looking at my desktop which runs KDE with KDEConnect, when I  noticed that I could pair it with my laptop which runs gnome with GSConnect.

I paired and noticed that on my desktop it lists a plugin called "Virtual Monitor".  I was wondering if GSConnect supports being a virtual monitor, as I couldn't find the option on my laptop, and couldn't find the button to actually start using it as a virtual monitor

**Comment by andyholmes** [comment]
There is no support for this plugin in GSConnect, sorry. As far as I know, no one has planned to implement it yet.

**Comment by ferdnyc** [comment]
If it helps any, the [most recent commit messsage](https://github.com/KDE/kdeconnect-kde/commit/2716a7a2e6583ead4cc4e9a9a9efb831dd7b28e7) for that plugin in the `kdeconnect-kde` repository (the one for the desktop version of KDEConnect), from just a week ago, is:


> ### Fixed virtualmonitorplugin url generation
> 
> BUG: 485830
> 
> The current implementation of the plugin is severly broken.
> 
> 1. The generated URL links to the localhost
> 2. The port is not set
> 
> -----
> 
> The URL is now generated on the request receiving side, not send in the request.
> This allows finding a valid IP address.
> 
> Furthermore, I changed the protocol by splitting it up. This could become useful, if we ever want to support other rdp protocols/platforms.
> 
> Note: This is a breaking change, but the current implementation is not working at all.. so it does not actually break something.

So it's debatable whether **`kdeconnect`** even supports Virtual Monitor, really. The plugin is apparently in an... "evolving" state.

**Reply by ferdnyc** [reply]
I decided to try and see if we could just use `krfb-virtualmonitor` to implement this, same as `kdeconnectd` does. Running it on my GNOME 46 / Fedora 40 VM to export the display seemed to work fine, but it immediately crashed the moment I connected `vncviewer` to it from the host (Fedora 39) system. :-1:
